server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/GroupWsSupport.java

```diff
@@ -0,0 +1,202 @@
+      organizationUuid = defaultOrganizationProvider.get().getUuid();
+    } else {
+      organizationUuid = findOrganizationByKey(dbSession, organizationKey).getUuid();
+    }
+    return GroupRef.fromName(organizationUuid, name);
+  }
+
+  /**
+   * Loads organization from database by its key.
+   * @param dbSession
+   * @param key the organization key, or {@code null} to get the default organization
+   * @return non-null organization
+   * @throws NotFoundException if no organizations match the provided key
+   */
+  OrganizationDto findOrganizationByKey(DbSession dbSession, @Nullable String key) {
+    String effectiveKey = key;
+    if (effectiveKey == null) {
+      effectiveKey = defaultOrganizationProvider.get().getKey();
+    }
+    Optional<OrganizationDto> org = dbClient.organizationDao().selectByKey(dbSession, effectiveKey);
+    checkFoundWithOptional(org, "No organization with key '%s'", key);
+    return org.get();
+  }
+
+  /**
+   * Similar to {@link UserGroupValidation#validateGroupName(String)} but kept internal. No need to publish
+   * this method in public API.
+   * @return the same description
+   */
+  @CheckForNull
+  String validateDescription(@Nullable String description) {
+    checkArgument(description == null || description.length() <= DESCRIPTION_MAX_LENGTH,
+      "Description cannot be longer than %s characters", DESCRIPTION_MAX_LENGTH);
+    return description;
+  }
+
+  void checkNameDoesNotExist(DbSession dbSession, String organizationUuid, String name) {
+    // There is no database constraint on column groups.name
+    // because MySQL cannot create a unique index
+    // on a UTF-8 VARCHAR larger than 255 characters on InnoDB
+    if (dbClient.groupDao().selectByName(dbSession, organizationUuid, name).isPresent()) {
+      throw new BadRequestException(format("Group '%s' already exists", name));
+    }
+  }
+
+  WsUserGroups.Group.Builder toProtobuf(OrganizationDto organization, GroupDto group, int membersCount) {
+    WsUserGroups.Group.Builder wsGroup = WsUserGroups.Group.newBuilder()
+      .setId(group.getId())
+      .setOrganizationKey(organization.getKey())
+      .setName(group.getName())
+      .setMembersCount(membersCount);
+    if (group.getDescription() != null) {
+      wsGroup.setDescription(group.getDescription());
+    }
+    return wsGroup;
+  }
+
+  static void defineWsGroupParameters(WebService.NewAction action) {
+    defineWsGroupIdParameter(action);
+    defineWsGroupNameParameter(action);
+  }
+
+  private static void defineWsGroupIdParameter(WebService.NewAction action) {
+    action.createParam(PARAM_GROUP_ID)
+      .setDescription("Group id")
+      .setExampleValue("42");
+  }
+
+  private static void defineWsGroupNameParameter(WebService.NewAction action) {
+    action.createParam(PARAM_ORGANIZATION_KEY)
+      .setDescription("Key of organization")
+      .setExampleValue("my-org")
+      .setSince("6.2");
+    action.createParam(PARAM_GROUP_NAME)
```

Comment:
no, as this parameter was created at the same time than the ws. Since fields are overridden only on new parameters.